tests/blackbox_integration.rs

```diff
@@ -97,6 +97,8 @@ impl TestCase {
 
         cmd.current_dir(self.test_dir.as_path())
             .arg("-s")
+            .arg("--log")
+            .arg("trace")
```

Comment:
Not sure we want this as a default.

Reply:
I used this for debugging, because the test output was kinda useless. Would be nice if one could change this from the outside. Still the output is hidden, unless the test fails, so the only tradeoff here is a little slower execution time of the tests. I am also fine removing these though.

Reply:
The test cases should inherit `RUST_LOG` from the environment, so setting that to `trace` should work as well (haven't tested it, though).

Reply:
Yes, I thought that too, but it does not work, because we use a constructor that ignores the environment.

Reply:
ah, all right then, let's leave this in